src/import/FSHImporter.ts

```diff
@@ -1232,6 +1232,8 @@ export class FSHImporter extends FSHVisitor {
     // Get the caret path, but slice off the starting ^
     caretValueRule.caretPath = this.visitCaretPath(ctx.caretPath()).slice(1);
     caretValueRule.value = this.visitValue(ctx.value());
+    caretValueRule.isInstance =
```

Comment:
I see that this is set here, but is this used anywhere?

Reply:
Nope! I think I was planning to use this as the condition for whether to put a rule into `deferredRules`, but maybe that's not necessary? I thought I needed it for the same reason that FixedValueRule needs it, but now I'm thinking about the case where an alias is used for the URL to an instance that will be present in the loaded FHIR definitions. So, in that case, this condition would actively work against me.